common/cuda_hip/matrix/csr_kernels.hpp.inc

```diff
@@ -1116,6 +1257,408 @@ void build_lookup(std::shared_ptr<const DefaultExecutor> exec,
 }
 
 
+namespace {
```

Comment:
those function without permute like spgeam or spgemm are only moved from cuda/hip to cuda_hip, right?

Reply:
yes